### PR TITLE
Updated pulp docs with explicit instructions

### PR DESCRIPTION
**Pull request by harshad16**
Updated pulp docs with explicit instructions
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Description

Added this , so we don't have to do this anymore: https://github.com/operate-first/support/issues/509#issuecomment-1015482126

### CODE REVIEW

File: usage.md [hunk 1]
```diff
@@ -18,16 +18,20 @@ Python package indexes that can be specific to teams or user needs.
...
 ## Requesting a Python package index on Operate First Pulp
 
+Request a repository on the Operate first Pulp Python package index for your package.
 [Create a request at
 operate-first/support](https://github.com/operate-first/support/issues/new/choose)
 using the "*New Python package index request*" template.
 
+Admin of the Operate first Pulp Python package index, would create the index
+and provide the url for the explicit package.
+
```
Comment (codificat) [inline usage.md]: Minor suggestion: put this as steps to follow:

```suggestion
1. Request a repository on the Operate first Pulp Python package index for your package by
[creating a request at
operate-first/support](https://github.com/operate-first/support/issues/new/choose)
using the "*New Python package index request*" template.

2. An admin of the Operate first Pulp Python package index will create the index
and provide the url for the explicit package in the ticket.

```

**Review by codificat — COMMENTED** [review]
/lgtm

only a minor suggestion to put the instructions as numbered steps

File: usage.md [hunk 2]
```diff
@@ -50,8 +54,23 @@ configuration](https://thoth-station.ninja/docs/developers/adviser/experimental_
...
+Follow instructions:
+
+- Install [aicoe-ci](https://github.com/apps/aicoe-ci) GitHub app in your GitHub repository.
+- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml).
+- Edit the below snippet with your explicit **package name** and add it in `.aicoe-ci.yaml` configuration yaml
+
+```yaml
+package-upload:
+  - instance: pulp
+    pulp-index: <package-name>
```
Comment (fridex) [inline usage.md]:
```suggestion
    pulp-index: <index-name>
```

File: usage.md [hunk 3]
```diff
@@ -50,8 +54,23 @@ configuration](https://thoth-station.ninja/docs/developers/adviser/experimental_
...
 Python packages to Pulp as Pulp does not support RBAC. Thus AICoE-CI acts as an
-authority for publishing Python packages to Pulp indexes. Follow instructions
-in
-[thoth-station/aicoe-ci-pulp-upload-example](https://github.com/thoth-station/aicoe-ci-pulp-upload-example)
+authority for publishing Python packages to Pulp indexes.
+Follow instructions:
+
+- Install [aicoe-ci](https://github.com/apps/aicoe-ci) GitHub app in your GitHub repository.
+- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml).
+- Edit the below snippet with your explicit **package name** and add it in `.aicoe-ci.yaml` configuration yaml
```
Comment (fridex) [inline usage.md]: An index can host multiple packages (I was surprised with the donkeycar name used):

```suggestion
- Edit the below snippet with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,9 @@
...
-2. Configuration for Coala and Black (code formating) are done using [`prow`](https://github.com/kubernetes/test-infra/tree/master/prow) with `pre-commit` and `mypy`.
-3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
-4. We have a public service configuration now accessible for everyone.
-5. If you are writing a Python module, [Kebechet](https://github.com/marketplace/khebhut) could manage the releases of your packages for you.
-6. If credentials are provided, Python module releases could be published to [`PyPI`](https://pypi.org/) by [Kebechet](https://github.com/marketplace/khebhut).
+## setup instruction
 
-Dependencies should be managed using `pipenv` (`Pipfile`, and the `Pipfile.lock` could be created by `thamos advise`), `pip3` and a `requirements.txt` files could be used.
+- Repo is listed as the source for the pulp documentation on operate-first site: <https://github.com/operate-first/operate-first.github.io/blob/90691bc8bc2eb4ae746d61d685b87a2da9c0ca3a/gatsby-config.js#L65>
+- On update of the [usage](./usage.md) doc the documentation get updated.
```
Comment (KPostOffice) [inline README.md]:
```suggestion
- On update of the [usage](./usage.md) doc the documentation gets updated.
```

File: usage.md [hunk 5]
```diff
@@ -50,8 +54,23 @@ configuration](https://thoth-station.ninja/docs/developers/adviser/experimental_
...
 Python packages to Pulp as Pulp does not support RBAC. Thus AICoE-CI acts as an
-authority for publishing Python packages to Pulp indexes. Follow instructions
-in
-[thoth-station/aicoe-ci-pulp-upload-example](https://github.com/thoth-station/aicoe-ci-pulp-upload-example)
+authority for publishing Python packages to Pulp indexes.
+Follow instructions:
+
+- Install [aicoe-ci](https://github.com/apps/aicoe-ci) GitHub app in your GitHub repository.
+- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml).
+- Edit the below snippet with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml
```
Comment (KPostOffice) [inline usage.md]:
```suggestion
- Edit the snippet below with your explicit **index name** and add it in `.aicoe-ci.yaml` configuration yaml
```

File: usage.md [hunk 6]
```diff
@@ -50,8 +54,23 @@ configuration](https://thoth-station.ninja/docs/developers/adviser/experimental_
...
+
+```yaml
+package-upload:
+  - instance: pulp
+    pulp-index: <index-name>
+    allow-sdist: true
+    allow-bdist: true
+```
+
+- Create a tag in GitHub and the app would upload the package to operate first pulp python index.
```
Comment (KPostOffice) [inline usage.md]:
```suggestion
- Create a tag in GitHub and the app will upload the package to operate first pulp python index.
```

**Review by KPostOffice — COMMENTED** [review]
A few tiny comments

File: usage.md [hunk 7]
```diff
@@ -50,8 +54,23 @@ configuration](https://thoth-station.ninja/docs/developers/adviser/experimental_
...
 Currently, only [AICoE-CI](https://github.com/AICoE/aicoe-ci) can publish
 Python packages to Pulp as Pulp does not support RBAC. Thus AICoE-CI acts as an
-authority for publishing Python packages to Pulp indexes. Follow instructions
-in
-[thoth-station/aicoe-ci-pulp-upload-example](https://github.com/thoth-station/aicoe-ci-pulp-upload-example)
+authority for publishing Python packages to Pulp indexes.
+Follow instructions:
+
+- Install [aicoe-ci](https://github.com/apps/aicoe-ci) GitHub app in your GitHub repository.
+- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml).
```
Comment (fridex) [inline usage.md]: The page links to the example repo in other places:

```suggestion
- Include `.aicoe-ci.yaml` configuration.yaml, a sample can be found [here](https://github.com/thoth-station/aicoe-ci-pulp-upload-example/blob/master/.aicoe-ci.yaml).
```

**Review by fridex — COMMENTED** [review]
Once comment, otherwise LGTM! 👍🏻

**Review by fridex — APPROVED** [review]
looks good to me, thanks! 💯 🥇

**Pull request merged**